pkg/theia/commands/anomaly_detection_run.go

```diff
@@ -240,5 +259,14 @@ Example values include 0.1, 500m, 1.5, 5, etc.`,
 		`Specify the memory request for the executor Pod. Values conform to the Kubernetes resource quantity convention.
 Example values include 512M, 1G, 8G, etc.`,
 	)
-
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"agg-flow",
+		"",
+		`Specifies which aggregated flow to perform anomaly detection on, options are pods/pod2pod/pod2svc`,
+	)
+	throughputAnomalyDetectionAlgoCmd.Flags().String(
+		"p2p-label",
+		"",
+		`On choosing agg-flow as pod2pod, user need to mention labels for inbound/outbound throughput`,
```

Comment:
For the newly added functionality, could you include them in documentation?

Reply:
Thanks for the comment, I have added it in the other PR [192](https://github.com/antrea-io/theia/pull/192/commits/7f037fac89ce35c1ff9a86f26b0a6b4db9239cfa#diff-30e6dc527a9063cf9b844f7002e3a7d11c6df87ebe5f67deaf16e706f6da2738)